server/src/internalClusterTest/java/org/opensearch/snapshots/DeleteSnapshotV2IT.java

```diff
@@ -32,7 +32,7 @@
 import static org.hamcrest.Matchers.lessThan;
 
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0)
-public class DeleteSnapshotITV2 extends AbstractSnapshotIntegTestCase {
+public class DeleteSnapshotV2IT extends AbstractSnapshotIntegTestCase {
```

Comment:
Adding a comment so that this PR is not merged before https://github.com/opensearch-project/OpenSearch/pull/15738